src/App.tsx

```diff
@@ -1,26 +1,41 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useState } from 'react';
+import { Errors } from './types/Errors';
+import { Filter } from './types/Filter';
+import { AppState } from './types/AppState';
+import { Notification } from './components/Notification/Notification';
+import { Footer } from './components/Footer/Footer';
+import { Header } from './components/Header/Header';
+import { Main } from './components/Main/Main';
 
-const USER_ID = 0;
+const INITIAL_STATE = {
+  todos: [],
+  error: Errors.noError,
+  filter: Filter.all,
+  loadingTodos: [],
+  tempTodo: null,
+  isEdited: false,
+};
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [appState, setAppState] = useState<AppState>(INITIAL_STATE);
 
   return (
-    <section className="section container">
-      <p className="title is-4">
-        Copy all you need from the prev task:
-        <br />
-        <a href="https://github.com/mate-academy/react_todo-app-add-and-delete#react-todo-app-add-and-delete">
-          React Todo App - Add and Delete
-        </a>
-      </p>
+    <div className="todoapp">
+      <h1 className="todoapp__title">todos</h1>
+
+      <div className="todoapp__content">
+        <Header appState={appState} updateState={setAppState} />
+
+        <Main appState={appState} updateState={setAppState} />
+
+        {!!appState.todos.length && (
+          <Footer appState={appState} updateState={setAppState} />
+        )}
```

Comment:
These components must be without props, fix it

Reply:
Done. Header, Main and Footer now without props.